app/controllers/vm_common.rb

```diff
@@ -539,7 +539,7 @@ def policy_options
 
   # Set right_size selected db records
   def right_size(record = nil)
-    @record ||= record ? record : find_records_with_rbac(params[:id]).first
+    @record ||= record ? record : find_record_with_rbac(params[:id])
```

Comment:
@PanSpagetka should this be `Vm` or `VmOrTemplate`, any idea? (https://github.com/ManageIQ/manageiq-ui-classic/commit/56724b1c#diff-5910cce3bec4cd624ba288de86215b4cR542)